tools/accuracy_checker/accuracy_checker/annotation_converters/cmu_panoptic.py

```diff
@@ -0,0 +1,118 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from pathlib import Path
+
+import cv2
+
+from ..utils import read_json, check_file_existence
+from ..representation import PoseEstimation3dAnnotation
+from .format_converter import DirectoryBasedAnnotationConverter, ConverterReturn
+
+
+class CmuPanopticKeypointsConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'cmu_panoptic_keypoints'
+    annotation_types = (PoseEstimation3dAnnotation,)
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        data = {}
+        images_dir_name = 'hdImgs'
+        labels_dir_name = 'hdPose3d_stage1_coco19'
+        label_prefix = 'body3DScene'
+        num_iterations = 0
+        for scene_dir in self.data_dir.iterdir():
+            data[scene_dir] = {}
+            scene_images_dir = scene_dir / images_dir_name
+            for camera_dir in scene_images_dir.iterdir():
+                data[scene_dir][camera_dir] = [file_path for file_path in camera_dir.iterdir()]
+                num_iterations += len(data[scene_dir][camera_dir])
+
+        keypoints_annotations = []
+        content_errors = []
+        image_id = 0
+        for scene_dir, camera_dir in data.items():
+            calibration_name = 'calibration_{}.json'.format(scene_dir.stem)
+            calibration = read_json(scene_dir / calibration_name)
+            for camera_dir, images_paths in camera_dir.items():
```

Comment:
you use the same name of variable in external and internal cycle (`camera_dir`). is it correct?